Library/Homebrew/test.rb

```diff
@@ -38,12 +38,13 @@
   formula.extend(Debrew::Formula) if args.debug?
 
   ENV.extend(Stdenv)
-  T.cast(ENV, Stdenv).setup_build_environment(formula: formula, testing_formula: true)
+  ENV.setup_build_environment(formula: formula, testing_formula: true)
 
   # tests can also return false to indicate failure
-  run_test = proc do
-    raise "test returned false" if formula.run_test(keep_tmp: args.keep_tmp?) == false
-  end
+  # TODO: the current version of sorbet does not support `proc` here
```

Comment:
What's the error here? Is there an issue we can link to here?

Reply:
I meant the `TODO` line of course, not the RuboCop line.

Reply:
No linked issue that I can find. Here's a summary of the distinction: https://sorbet.run/#%23%20typed%3A%20true%0A%0Arequire%20'timeout'%0A%0Ap1%20%3D%20Proc.new%20%7B%20raise%20%22p1%20large%20rand%22%20if%20rand%20%3E%200.5%7D%0AT.reveal_type%28p1%29%0Ap2%20%3D%20proc%20%7B%20raise%20%22p2%20large%20rand%22%20if%20rand%20%3E%200.5%7D%0AT.reveal_type%28p2%29%0A%0ATimeout.timeout%281%2C%20%26p1%29%0ATimeout.timeout%281%2C%20%26p2%29

It could probably be avoided with a bit of redirection, but i'd prefer to disable the cop:

https://sorbet.run/#%23%20typed%3A%20true%0Aextend%20T%3A%3ASig%0A%0Arequire%20'timeout'%0A%0Asig%20do%0A%20%20params%28%0A%20%20%20%20p1%3A%20Proc%2C%0A%20%20%20%20p2%3A%20T.proc.returns%28Integer%29%0A%20%20%29%0A%20%20.void%0Aend%0Adef%20example%28p1%2C%20p2%29%0A%20%20Timeout.timeout%281%2C%20%26p1%29%0A%20%20Timeout.timeout%281%2C%20%26p2%29%0Aend%0A

wdyt?

Reply:
Okay, so it's because the `proc` doesn't have an argument?